views/pages/EditConfigPage.jsx

```diff
@@ -66,20 +64,29 @@ const EditConfigPage = () => {
   const onCopy = (configCategoryIndex) =>
     addNewField(getValues(`config[${configCategoryIndex}]`))
 
+  useEffect(() => {
+    fetchCurrentConfig().then((values) => {
+      reset(values)
+      setLoading(false)
+    })
+  }, [])
+
   if (loading) return <div>Loading...</div>
 
   return (
-    <ConfigForm
-      colors={colors}
-      control={control}
-      fields={fields}
-      friendsList={friendsList}
-      gridState={gridState}
-      onAddNewField={addNewField}
-      onCopy={onCopy}
-      onSubmit={handleSubmit(handleFormData)}
-      onRemove={removeField}
-    />
+    <Box sx={{ p: '30px' }}>
+      <Typography variant="h6">Edit Configuration</Typography>
```

Comment:
This is just an aside but WTF is wrong with an `h6` tag `@mui`?

Reply:
haha, yeah it's weird with the variants